lib/awskeyring.rb

```diff
@@ -39,82 +52,95 @@ def self.load_keychain
     keychain
   end
 
+  # Return a list of all acount items
   def self.list_items
     items = all_items.all.sort do |a, b|
       a.attributes[:label] <=> b.attributes[:label]
     end
     items.select { |elem| elem.attributes[:label].start_with?(ACCOUNT_PREFIX) }
   end
 
+  # Return a list of all role items
   def self.list_roles
     items = all_items.all.sort do |a, b|
       a.attributes[:label] <=> b.attributes[:label]
     end
     items.select { |elem| elem.attributes[:label].start_with?(ROLE_PREFIX) }
   end
 
+  # Return all keychain items
   def self.all_items
     load_keychain.generic_passwords
   end
 
+  # Add an account item
   def self.add_item(account:, key:, secret:, comment:)
     all_items.create(
-      label: "#{ACCOUNT_PREFIX}#{account}",
+      label: ACCOUNT_PREFIX + account,
```

Comment:
For what reason did you decide to change interpolation?

Reply:
it seemed a bit silly to use interpolation when there was only two strings and no other formatting.